app/assets/javascripts/converse.js

```diff
@@ -0,0 +1,23 @@
+//= require converse.js.js
+
+// initialize converse xmpp client
+$(document).ready(function() {
+  if (app.currentUser.authenticated()) {
+    $.post("/user/auth_token", null, function(data) {
+      if (converse && data.token) {
+        converse.initialize({
+            bosh_service_url: $('script#converse').data('endpoint'),
```

Comment:
- Expected indentation of 10 space characters but found 12.
- Identifier 'bosh_service_url' is not in camel case.
- Strings must use doublequote.